packages/calcite-design-tokens/support/types/styleDictionary/dictionary.ts

```diff
@@ -0,0 +1,12 @@
+import { Dictionary as SdDictionary } from "style-dictionary/types/Dictionary.js";
+import { TransformedToken } from "./transformedToken";
+import { TransformedTokens } from "./transformedTokens";
+
+export type Dictionary = SdDictionary & {
+  allTokens: TransformedToken[];
+  tokens: TransformedTokens;
+  allProperties: TransformedToken[];
+  properties: TransformedTokens;
+  usesReference: (value: any) => boolean;
```

Comment:
Can the type be narrowed or is it unknown?

Reply:
It could potentially be anything at this point.